external/ftpc/ftpc_utils.c

```diff
@@ -237,6 +237,9 @@ FAR char *ftpc_dequote(FAR const char *str)
 				if (str[0] == '%') {
 					/* Extract the hex value */
 
+					if (!str[1] || !str[2]) {
+						break;
```

Comment:
I think that you use 'len' because str[1] and str[2] can point invalid memory space.

Reply:
@JeonginKim  if the input buffer is terminated by  character 0 than checking str[1] and str[2] would be enough. I will merge it